models/twilio_message_enhanced.sql

```diff
@@ -57,12 +57,12 @@ outbound_messages as (
     where direction like '%outbound%'
 ),
 
-union_messages as (
-
-    select * from inbound_messages
-    union all
-    select * from outbound_messages
-),
+union_messages as ( 
+ 
+    select * from inbound_messages 
+    union all 
+    select * from outbound_messages 
+), 
```

Comment:
Can we move everything up to the union step to its own intermediate model.

Reply:
yeah, thought about making messages into it's own intermediate... moved